packages/examples/examples/typescript/package.json

```diff
@@ -15,6 +15,7 @@
     "snap.manifest.json"
   ],
   "scripts": {
+    "test:e2e": "jest --runInBand",
```

Comment:
I need to figure out if we can set this from within the environment, or if there's a better way around this.